code/datums/uplink/devices and tools.dm

```diff
@@ -89,12 +89,12 @@
 
 /datum/uplink_item/item/tools/thermal
 	name = "Thermal Imaging Glasses"
-	item_cost = 6
+	item_cost = 2
 	path = /obj/item/clothing/glasses/thermal/syndi
 
 /datum/uplink_item/item/tools/powersink
 	name = "Powersink (DANGER!)"
-	item_cost = 10
+	item_cost = 3
```

Comment:
```suggestion
	item_cost = 6
```
Depending on where it is placed it is very difficult to almost impossible to find.
So it should require a substantial investment.